conans/model/info.py

```diff
@@ -513,26 +434,28 @@ def package_id(self):
         options and settings
         """
```

Comment:
just to make sure, this is going to change the package_id for the currently built packages (in the v2 repository), right?

Reply:
Most likely not, as it didnt change any package_id in the codebase. This change only kicks in when building executables, defined with the ``package_type = "application"``. If I understood correctly, @czoido is faking the cmake package build at the moment, so shouldn't be an issue.